applets/notification_area/na-grid.c

```diff
@@ -54,7 +54,7 @@ enum
   PROP_ICON_SIZE
 };
 
-G_DEFINE_TYPE (NaBox, na_box, GTK_TYPE_BOX)
+G_DEFINE_TYPE (NaGrid, na_grid, GTK_TYPE_BOX)
```

Comment:
Should not this be `GTK_TYPE_GRID` here? I see that you changed that in next commit, but it should be in this commit...

Reply:
Well this looks incomplete to me. If it was supposed to be still `GtkBox` then why did you change `GtkBox parent;` to `GtkGrid parent;`? Sound that in this commit it should be `GtkBox parent;` still.